Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/FitGaussianPeaksTest.py

```diff
@@ -570,14 +573,15 @@ def test_algorithm_estimates_fit_window(self):
                              PeakGuessTable=table,
                              EstimateFitWindow=True,
                              FitWindowSize=11)
-            correct_x_val = np.array([0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22])
-            correct_y_val = np.array([0, 0, 0, 0, 1, 7, 0, 0, 0, 0, 10, 7])
 
+            centre_index = 10
+            # win_size is ( FitWindowSize -1)/2 as method estimate_single_parameters expects in this form
```

Comment:
it says fitwindowsize is 11, so should win size be (11-1)/2 = 5?

Reply:
I forgot to put this in the comments but the algorithm is running as it was originally coded to calculate win_size from EstimatePeakSigma so it will be compatible with algorithms that originally use FitGaussianPeaks and calculated value for win_size is 2

Reply:
giving it an argument and then not using it is very confusing. If I provide a value, I expect that to be used

Reply:
This test is to check if it ignores that value that is why a different arguement is provided.

Reply:
It should be clear from the inputs in the function that it will be ignored, highlighted in the comment along with what value you expect instead. However, we should avoid ignoring user input

Reply:
It is in the comments that FitWindowSize is ignored and it is only ignored when EstimateFitWindow is True so users can choose if they want to directly specify the fit window size